index.js

```diff
@@ -1,5 +1,8 @@
 module.exports = (nextConfig = {}) => {
   return Object.assign({}, nextConfig, {
+    publicRuntimeConfig: Object.assign({}, nextConfig.publicRuntimeConfig, {
+      nextImagesAssetPrefix: nextConfig.assetPrefix
+    }),
     webpack(config, options) {
       const { isServer } = options;
       nextConfig = Object.assign({ inlineImageLimit: 8192, assetPrefix: "" }, nextConfig);
```

Comment:
This is outdated. Please rebase this branch with master branch

Reply:
I rebased my branch on your master branch.